src/views/LimitedRestorationExtension.vue

```diff
@@ -1,14 +1,27 @@
 <template>
   <ViewWrapper>
-    <section class="pb-10" id="restoration-view">
+    <section class="pb-10" id="limited-restoration-extension">
       <!-- Company Information page-->
       <v-slide-x-transition hide-on-leave>
-        <div v-if="!isSummaryMode">
+        <div v-if="!isSummaryMode" id="question_container">
           <header>
-            <h1>{{ entityTitle }}</h1>
+            <h1>Limited Restoration Extension</h1>
           </header>
+          <QuestionWrapper
+            id="applicant-information"
+            title="Applicant Information"
+            subtitle="You application must include one of the following"
+          >
+            <PeopleAndRoles />
+          </QuestionWrapper>
 
-          <PeopleAndRoles class="mt-10" />
+          <QuestionWrapper
+            id="extend-time-limit"
+            title="Extend Time Limit of Limited Restoration"
+            subtitle="Select an extension time"
```

Comment:
Also add colon at end.

Reply:
The uxpin is inconsistent here: for question 1 the subtitle is bolded, but for question 2 it isn't.

Please follow up with Ethan to fix the uxpin.

Re: https://preview.uxpin.com/306da47387722817e24fc77fd71476a1869a05fe#/pages/160258061/comment/sitemap

Reply:
Yes, I wondered about this too, but I went back through other Uxpins and found a number of other examples of some subtitles bolded and some subtitled not bolded.  I also tried changing the bolded subtitle and it doesn't look right.  If it's okay with you, I'd like to leave it as is.  If there's a design change in the future, we can fix it then.

Reply:
Or, you could use the "comment" prop to display the text non-bolded. Up to you, but there's a mismatch between the uxpin and this code.

Reply:
OK with me to leave it.